fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/SplitHelperTest.java

```diff
@@ -121,13 +123,16 @@ public void setSubspace() {
     }
 
     @Nonnull
-    public static Stream<Arguments> splitAndSuffixArgs() {
-        // Arguments: splitLongRecords, omitUnsplitSuffix
-        // Note that "true", "true" is not valid
+    public static Stream<Arguments> splitSuffixAndUnrollArgs() {
```

Comment:
This might be more readable with a `flatMap`, e.g.
```
Stream.of(false, true).flatMap(splitLongRecords -> 
    (splitLongRecords ? Stream.of(false) : Stream.of(false, true)).flatMap(omitUnsplitSuffix -> 
        Stream.of(false, true).map(unrollSingleRecordDeletes -> Arguments.of(splitLongRecords, omitUnsplitSuffix, unrollSingleRecordDeletes))))
```